.github/workflows/e2eLocal.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         CYPRESS_OKTA_PASSWORD: ${{ secrets.CYPRESS_OKTA_PASSWORD }}
         CYPRESS_OKTA_SECRET: ${{ secrets.CYPRESS_OKTA_SECRET }}
         WIREMOCK_URL: "http://cypress:8088"
-        SPEC_PATH: "cypress/integration/*"
+        SPEC_PATH: "cypress/integration/01*"
```

Comment:
This line tells the ci to only run this test, this should be removed before merging.